meson_options.txt

```diff
@@ -147,6 +147,18 @@ option('system-uid-max', type : 'string',
        description : 'maximum system UID')
 option('system-gid-max', type : 'string',
        description : 'maximum system GID')
+option('dynamic-uid-min', type : 'string',
+       description : 'minimum dynamic UID',
+       value : '61184') # That's → 0x0000EF00 in hex
```

Comment:
Why not just say `value : '0xEF00'`?

Reply:
because meson doesn't support that... it uses python's int() call to convert things to integers, and that can only decode regarding base 10. I figure we should file a bug against meson asking for them to support to base 16 decoding too, with 0x prefixing, but I was lazy so far

Reply:
Right. There's a PR open: https://github.com/mesonbuild/meson/pull/2061, I thought it went in, but it did not. Either way, we cannot use hex for now.